blob/blob.go

```diff
@@ -63,8 +65,17 @@ type Blob struct {
 }
 
 // NewBlob constructs a new blob from the provided namespace.ID and data.
-func NewBlob(_ uint8, nID namespace.ID, data []byte) (*Blob, error) {
-	blob, err := types.NewBlob(nID, data)
+func NewBlob(shareVersion uint8, nID namespace.ID, data []byte) (*Blob, error) {
```

Comment:
[optional] I've been using the following terminology to avoid confusion when referring to namespace variables:

version + ID = namespace

or longer form:

namespace version + namespace ID = namespace

so in this case, the second parameter is the entire namespace because it contains the version

```suggestion
func NewBlob(shareVersion uint8, namespace namespace.ID, data []byte) (*Blob, error) {
```

Reply:
That's helpful. Thank you!